griptape/artifacts/json_artifact.py

```diff
@@ -7,11 +7,11 @@
 
 from griptape.artifacts import BaseArtifact
 
-Json = Union[dict[str, "Json"], list["Json"], str, int, float, bool, None]
-
 
 @define
 class JsonArtifact(BaseArtifact):
+    Json = Union[dict[str, "Json"], list["Json"], str, int, float, bool, None]
+
```

Comment:
In the near future we may offer some sort of structured output functionality that would allow users to pass in a JSON schema they'd like the LLM to output their answer in.

The type could be something like: `output_format: JsonArtifact.Json`